libwallet/assets/btc/rescan.go

```diff
@@ -390,28 +389,3 @@ func (asset *Asset) getblockStamp(height int32) (*waddrmgr.BlockStamp, error) {
 		Timestamp: block.Header.Timestamp,
 	}, nil
 }
-
-// updateSyncedToBlock is used to update syncedTo block. Sometimes btcwallet might
-// miss the trigger event to update syncedTo block so the update is done here
-// regardless thus avoid handling the possible scenario where btcwallet might miss
-// the syncedto store trigger event.
-func (asset *Asset) updateSyncedToBlock(height int32) {
```

Comment:
why remove this function, it need to update progress bar for UI

Reply:
I don't think that is what it does. It is there because sometimes btcwallet was not getting the notifications it needed, so this was a bandaid.